packages/modules/src/modules/banners/choiceCardsButtonsBanner/components/ChoiceCards.tsx

```diff
@@ -80,7 +79,10 @@ const styles = {
             border-radius: 10px;
         }
         legend {
-            ${visuallyHidden};
+            opacity: 0;
+            height: 0;
+            width: 0;
+            left: 0;
```

Comment:
This definition of `visuallyHidden` comes from the version of `@guardian/src-foundation` that we’re using, which is 3.8 – [here it is in the git history](https://github.com/guardian/source/blob/810897b334c050edcb0f1166ff2f16d51a2862d7/src/core/foundations/src/accessibility/index.ts#L4-L9) (though I see you’ve removed `position: absolute; top: 0;`, presumably to fix the layout issue you mentioned). However, they’ve [since been updated](https://github.com/guardian/source/pull/1551#pullrequestreview-1123815718) based on these guides:

- https://webaim.org/techniques/css/invisiblecontent/
- https://medium.com/@jessebeach/beware-smushed-off-screen-accessible-text-5952a4c2cbfe
- https://www.scottohara.me/blog/2017/04/14/inclusively-hidden.html

I worry that splitting off from the imported styles will mean these never get updated to the newer rules: could we use the newer rules here since we’re dropping the import anyway?

The [newest version of the rules](https://github.com/guardian/csnx/blob/1615f705d5151dd51cb51a9a37bd20eca59c1417/libs/@guardian/source-foundations/src/accessibility/visually-hidden.ts#L11-L21) looks like this:

```css
	position: absolute;
	overflow: hidden; /* gets rid of horizontal scrollbar that appears in some circumstances */
	white-space: nowrap; /* The white-space property forces the content to render on one line. */
	width: 1px;  /* ensures content is announced by VoiceOver. */
	height: 1px; /* ensures content is announced by VoiceOver. */
	margin: -1px; /* hide or clip content that does not fit into a 1-pixel visible area. */
	padding: 0; /* hide or clip content that does not fit into a 1-pixel visible area. */
	border: 0;
	clip: rect(1px, 1px, 1px, 1px); /* clip removes any visible trace of the element */
	-webkit-clip-path: inset(50%); /* clip removes any visible trace of the element */
	clip-path: inset(50%); /* clip removes any visible trace of the element */
```

Presumably this text is meant to be visually hidden but still accessible to screen readers – have you verified that the screen reader can still read it with your changes? (This is probably more important than using an outdated approach.)

Reply:
Thats a very good spot, I did'nt think to look to see if the mew revised version of  '@guardian/src-foundations/accessibility' would have a later update. Think the 1px changes now allow a screenreader to still pickup the legend. Have copied the newest version of the rules here but perhaps another PR to upgrade to latest csnx source version might be worth putting backlog (suspect it would affect quite a bit perhaps?). Anyway, thanks, updated here for this PR a least.